.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   test-ruby-2-4-x:
-    runs-on: ubuntu-latest
```

Comment:
why is this change needed?

Reply:
isn't [this](https://github.com/fastruby/skunk/actions/runs/4845701187/jobs/8634780556?pr=109) the issue related to the ubuntu version?